### DISCUSSION TITLE
Looking for a good Next.js example using Bootstrap

### DISCUSSION THREAD

**Discussion by eusholli** [vercel/next.js#13255] [Help]
Hi, just completed next.js tutorial and deployed on vercel.com Thought the experience was excellent. I am now looking for a good example of a Next.js app using Bootstrap4.  I appreciate any suggestions

**Comment by helloworld** [comment]
There's an official example that uses `react-bootstrap`: https://github.com/zeit/next.js/tree/canary/examples/with-react-bootstrap

If you don't want to use the `react-bootstrap` components and just want to import the CSS, you can just install the `bootstrap` package and import the CSS the same way the example does here: https://github.com/zeit/next.js/blob/canary/examples/with-react-bootstrap/style/index.css
